src/gatsby/createPages/createDevelopmentApiPages.ts

```diff
@@ -0,0 +1,76 @@
+import { getDataOrPanic } from "./helpers";
+
+export default async ({ actions, graphql, reporter }) => {
+  const data = await getDataOrPanic(
+    `
+      query {
+        allOpenApi {
+          nodes {
+            id
+            paths {
+              description
+              method
+              operationId
+              summary
+              tags
+              apiPath
+              readableUrl
+              parameters {
+                schema {
+                  type
+                  format
+                  enum
+                }
+                name
+                in
+                description
+                required
+              }
+              responses {
+                content {
+                  content_type
+                  example
+                  schema
+                }
+                description
+                status_code
+              }
+              requestBody {
+                content {
+                  content_type
+                  schema
+                }
+                required
+              }
+              summary
+              tags
+              security {
+                auth_token
+              }
+            }
+          }
+        }
+      }
+    `,
+    graphql,
+    reporter
+  );
+
+  const component = require.resolve(`../../templates/developmentAPI.js`);
+  await Promise.all(
+    data.allOpenApi.nodes.map(async node => {
+      await Promise.all(
+        node.paths.map(async p => {
+          actions.createPage({
+            path: `/development-api/${p.readableUrl}`,
+            component,
+            context: {
+              ...p,
```

Comment:
its possible this _might_ balloon gatsbys stuff and make it angry

generally, and ive been learning this as im overhauling platforms, its recommended to create a Node that contains this data, and then read that node in the template itself otherwise you end up adding all the fields to every node of the same type (so now all pages have all these API attributes as context)

Reply:
ahh interesting! ok I'll make the necessary changes.